lib/src/views/Calendar/MonthSelection.tsx

```diff
@@ -77,17 +79,16 @@ export const MonthSelection: React.FC<MonthSelectionProps> = ({
         const monthNumber = utils.getMonth(month);
         const monthText = utils.format(month, 'monthShort');
 
-        return (
-          <Month
-            key={monthText}
-            value={monthNumber}
-            selected={monthNumber === currentMonth}
-            onSelect={onMonthSelect}
-            disabled={shouldDisableMonth(month)}
-          >
-            {monthText}
-          </Month>
-        );
+        const monthProps = {
+          key: monthText,
+          value: monthNumber,
+          selected: monthNumber === currentMonth,
+          onSelect: onMonthSelect,
+          disabled: shouldDisableMonth(month),
+          children: monthText,
+        };
+
+        return renderMonth ? renderMonth(month, monthProps) : <Month {...monthProps} />;
```

Comment:
Please keep the API consistant:

```suggestion
        return renderMonth ? renderMonth(monthProps) : <Month {...monthProps} />;
```